README.rst

```diff
@@ -0,0 +1,263 @@
+h1-cli
+======
+
+h1-cli jest konsolowym narzędziem przeznaczonym do zarządzania infrastrukturą chmury HyperOne. Możesz go wykorzystywać zarówno do własnych prac administracyjnych, jak również podczas tworzenia  skryptów automatyzujących.
+
+Przegląd funkcjonalności
+------------------------
+
+* kontrola wszystkich zasobów udostępnianych przez panel administracyjny, w szczególności:
+
+  * zmiana konfiguracji serwerów - ich tworzenie, wyłączanie i kasowanie,
+  * manipulacja adresami IP - ich przepinanie, modyfikacja rekordu PTR,
+  * modyfikowanie dysków - utworzenie, odłączenie / dołączenie od serwera, rozszerzenie, wykonanie migawki.
+
+* różnorodne formy uwierzytelniania,
+* wybór formatu wyjścia, w celu wykorzystania go w dodatkowych narzędziach.
+
+Instalacja
+----------
+
+Instalacja zasadniczo ogranicza się pobrania wykonywalnej kopii oprogramowania, która jest dostępna w zakładce "`Releases`_" repozytorium.
+
+W środowisku systemu Linux
+##############################
+
+Uruchom dostarczony plik binarny po nadaniu uprawnień wykonywalnych::
+
+    $ chmod +x h1_client.bin
+    $ ./h1_client.bin
+
+Aplikacja powinna działać poprawnie bez instalacji dodatkowych bibliotek / pakietów.
+
+Jeżeli zamierzasz z oprogramowania stale korzystać możesz dokonać instalacja dla użytkownika lub na poziomie systemu.
+
+Instalacja dla użytkownika może zostać przeprowadzona poprzez::
+
+   $ cp h1_client.bin ~/.local/bin/h1
```

Comment:
Jesteś pewien, że ~/.local jest wspierany przez większość znanych dystrybucji Linuxa ?

Reply:
Przedstawiam także alternatywę.

Zastanawiam się jak to sprawdzić…Skoro w "h1 vm ssh" wywołuje normalnie ssh ("spawn") to zastanawiam się dlaczego nie pozwolicie na przekazanie dowolnych dodatkowych argumentów do ssh. Wtedy byłoby mi łatwiej w pętli stworzyć VMki na różnych obrazach i sprawdzić domyślny $PATH.